doc/GettingStarted.md

```diff
@@ -122,6 +125,63 @@ In addition, Vitess requires the software and libraries listed below.
     ``` sh
     $ sudo apt-get install openjdk-7-jre
     ```
+    
+#### OSX
+
+1.  [Install Homebrew](http://brew.sh/).
+
+2.  On OSX, Mysql 5.6 has to be used, MariaDB doesn't work for some reason yet. It should be installed from Homebrew
+    (install steps are below).
+   
+3.  ZooKeeper is used as lock service.
+
+4.  Run the following commands:
+
+    ``` sh
+    $ brew install go automake libtool memcached python mercurial git bison curl wget homebrew/versions/mysql56
```

Comment:
I know we've been inconsistent elsewhere in this doc, but I think since this part is likely to be copy-pasted by the user, let's leave off the "$" prompts here.